### ISSUE TITLE
can't resolve enum

### ISSUE THREAD

**Issue by chrisdrackett** [mobxjs/mst-gql#29]
I have some enums in my types and after generating my models I get the following error:

```
/@calm/core/src/models/ItemModel.base.js
Module not found: Can't resolve './ItemTypeEnum.base' in '/@calm/core/src/models'
```

In ItemModel.base.js I see an unused import:

```
import { ItemTypeEnumSelector } from './ItemTypeEnum.base'
```

I wonder if this is suppose to be used below in this file under `ItemModelSelector`?

Sorry if this is related to #25 I'm still figuring everything out!

**Comment by dpnolte** [comment]
@chrisdrackett thanks for reporting! I also had the same issue: the import paths for enums were incorrectly generated. In your case, the import path should probably have been './ItemTypeEnum'.  Can you check if this is still the case? This should be fixed with #23

**Comment by chrisdrackett** [comment]
Yep! this is working now 🎉
